src/geoaxis.jl

```diff
@@ -0,0 +1,107 @@
+"""
+    GeoAxis(args...; kwargs...) → ax
+Create a new axis instance `ax` that is a modified `Axis` of the Makie.jl ecosystem.
+All Makie.jl plotting functions work directly on `GeoAxis`, e.g., `scatter!(ax, x, y)`.
+
+`GeoAxis` is appropriate for geospatial plotting because it automatically transforms all
+plotted data given a user-defined map projection. See keyword arguments below and examples
+in the online documentation. Longitude and latitude values in GeoMakie.jl are always
+assumed to be **in degrees**.
+
+In the call signature, `args...` is a standard figure location, e.g., `fig[1,1]` as given in
+`Axis`. The keyword arguments decide the geospatial projection:
+
+* `source = "+proj=longlat +datum=WGS84", dest = "+proj=eqearth"`: These two keywords
+  configure the map projection to be used for the given field using Proj4.jl.
+  See also online the section [Changing central longitude](@ref) for data that may not
+  span the (expected by default) longitude range from -180 to 180.
+* `transformation = Proj4.Transformation(source, dest, always_xy=true)`: Instead of
+  `source, dest` you can directly use the Proj4.jl package to define the projection.
+  lines!(ax, GeoMakie.coastlines(); coastkwargs...)
+* `lonticks = -180:60:180, latticks = -90:30:90` ticks for the longitude and latitude
+  dimensions. The grid lines of the axis are also spanning these tick values.
+* `hidespines = true` Hide the axis spines (rectangle surrounding the axis).
+
+## Example
+```julia
+using GeoMakie, GLMakie
+
+lons = -180:180
+lats = -90:90
+field = [exp(cosd(l)) + 3(y/90) for l in lons, y in lats]
+
+# Plot coastlines
+coastplot = lines!(ax, GeoMakie.coastlines(); color = :black, overdraw = true, coastkwargs...)
```

Comment:
`coastkwargs` is not defined
```suggestion
coastplot = lines!(ax, GeoMakie.coastlines(); color = :black, overdraw = true)
```

Reply:
Also, `ax` is not defined